gslb/nodes/avi_model_nodes.go

```diff
@@ -129,9 +130,14 @@ type PathHealthMonitorDetails struct {
 	Name            string
 	IngressProtocol string
 	Path            string
+	RequestHeader   string
+	ResponeCodes    []string
 }
 
-func (pathHm PathHealthMonitorDetails) GetPathHMDescription(gsName string) string {
+func (pathHm PathHealthMonitorDetails) GetPathHMDescription(gsName string, template *string) string {
+	if template != nil {
+		return CreatedByAMKO + ", gsname: " + gsName + ", path: " + pathHm.Path + ", protocol: " + pathHm.IngressProtocol + ", " + gslbutils.CreatedFrom + *template
```

Comment:
We have a repeated substring at line 139 and 141. We can combine this and add an append for template as this would reduce code maintenance.

Reply:
I have made the changes as per the comment.